syntaxes/graphql.connectors.yaml

```diff
@@ -0,0 +1,167 @@
+scopeName: source.apollo.connectors.graphql
+# inject these rules into source.graphql#graphql-directive, before all other patterns (:L), ignoring strings and comments
+injectionSelector: L:meta.embedded.block.directive.graphql -string -comment
+patterns:
+  - include: "#connect-directive"
+repository:
+  # if we encounter a `@connect` directive, move forward into this set of rules, otherwise it will just follow the default graphql rules
```

Comment:
I added a good bunch of comments to this file, please let me know if they are helpful or if I could make something more clear here.